tests/FunctionalTest.php

```diff
@@ -188,4 +188,60 @@ public function should_pass_request_object_based_on_type_hint()
         $response = $app->handle(Request::create('/?name=john'));
         $this->assertEquals('Hello john', $response->getContent());
     }
+
+    /**
+     * @test
+     */
+    public function should_able_to_convert_request()
```

Comment:
see above: I think it reads better with `should_be_able` for all of the new tests